.github/workflows/test.yml

```diff
@@ -5,11 +5,12 @@ name: Verify
 
 # Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the main branch
+  # Triggers the workflow on either push to main, or pull requests to any branch
```

Comment:
Do we need to indicate on readme this change, so contributors are aware? Or does that not matter?

Reply:
I... don't think so. I think if we setup the Actions "well enough", they should appear to "just work"

Contributors shouldn't have to worry about them until or unless they fail, and so I don't think it's worth being in the README. However, we do still have an issue open to add a CONTRIBUTING.md, and while info about Actions may or may not be necessary, things about linting or commit naming are definitely worthwhile - see #63